src/Store.ts

```diff
@@ -236,14 +236,14 @@ export interface HistoryState {
 }
 
 export type HistoryAction = {
-    type: 'Receive_Message' | 'Send_Message' | 'Show_Typing' | 'Receive_Sent_Message'
+    type: 'Receive_Message' | 'Send_Message' | 'Show_Typing' | 'Receive_Sent_Message',
```

Comment:
This is a good fix but it doesn't belong with this PR. Should be a separate "cleanup" PR so that the diffs are clear on what exactly is involved to the task of setting history.